src/KeyStoreUtils.ts

```diff
@@ -57,8 +59,16 @@ export namespace KeyStoreUtils {
             if (!bip39.validateMnemonic(mnemonic)) { throw new Error('The given mnemonic could not be validated.'); }
         }
 
-        const seed = (await bip39.mnemonicToSeed(mnemonic, password)).slice(0, 32);
-        const keys = await generateKeys(seed);
+        let keys: { secretKey: Buffer, publicKey: Buffer };
+        const seed = await bip39.mnemonicToSeed(mnemonic, password);
```

Comment:
How come we don't slice anymore?

Reply:
we still do, line 69, but that's how it works for non-derivation-path keys